packages/cra-template-mephisto-review/template/src/AllDataView.jsx

```diff
@@ -3,13 +3,19 @@ import { Redirect } from "react-router-dom";
 import { useMephistoReview } from "mephisto-review-hook";
 import { H4, H3, H2, H1, InputGroup, Button } from "@blueprintjs/core";
 import { Tooltip2 } from "@blueprintjs/popover2";
-import GridView from "./GridView";
+
+import GridView from "./components/GridView";
 import Pagination from "./components/Pagination";
-import "./GridRenderer.css";
+import "./css/allDataView.css";
+import DefaultItemRenderer from "./components/DefaultItemRenderer";
 
-function GridRenderer() {
-  const resultsPerPage = 9;
-  const [page, setPage] = useState(1);
+function AllDataView({
```

Comment:
We're being a little inconsistent here between how we use Data and Item. I think it makes more sense here to call this AllItemsView so it plays well with the other ItemView. Other alternatives: ItemCollectionView, ListItemView.

Reply:
Good catch, naming definitely did turn out to be the hardest part of this PR after all 😅